### PR TITLE
fixed incorrect journal editing

### PR DESCRIPTION
**Pull request by jimhavrilla**
the journal editor himself told us this was wrong and I fixed the CSL to amend this so here you go.

### CODE REVIEW

**Comment by jimhavrilla** [comment]
hopefully this helps users of Zotero/Paperpile like myself.

**Comment by jimhavrilla** [comment]
Looks like it may require some fixes for books.  Hopefully someone takes
the torch on that.

**Comment by jimhavrilla** [comment]
Still may need book fix

File: nucleic-acids-research.csl [hunk 1]
```diff
@@ -42,15 +45,15 @@
     <text variable="title"/>
   </macro>
   <macro name="publisher">
-    <group delimiter=", ">
-      <text variable="publisher"/>
-      <text variable="publisher-place"/>
-    </group>
+    <text variable="publisher"/>
+    <text variable="publisher-place"/>
   </macro>
```
Comment (rmzelle) [inline nucleic-acids-research.csl:50]: @POBrien333, would you mind taking a look at this style? This change in particular can't be right (no delimiter). See https://academic.oup.com/nar/pages/Ms_Prep_Submission#preparation for the current documentation link.

Comment (POBrien333) [reply nucleic-acids-research.csl:50]: Correct. 
This should be:
`  <macro name="publisher">
    <group delimiter=", ">
      <text variable="publisher"/>
      <text variable="publisher-place"/>
    </group>
  </macro>`

File: nucleic-acids-research.csl [hunk 2]
```diff
@@ -13,24 +13,27 @@
...
+      <name initialize-with="." name-as-sort-order="all" sort-separator=" " delimiter=", " delimiter-precedes-last="always"/>
       <label form="short" prefix=" (" suffix=")" strip-periods="true"/>
     </names>
   </macro>
   <macro name="author">
     <names variable="author">
-      <name name-as-sort-order="all" sort-separator="," initialize-with="." delimiter=", " and="text" delimiter-precedes-last="never"/>
-      <et-al font-style="italic"/>
+      <name name-as-sort-order="all" sort-separator=" " initialize-with="." delimiter=", " and="text" delimiter-precedes-last="never"/>
+      <et-al/>
```
Comment (POBrien333) [inline nucleic-acids-research.csl:36]: this line is obsolete now.

**Comment by adam3smith** [comment]
@jimhavrilla I'm confused by this:
Their instructions for authors have:
> Schmitt,E., Panvert,M., Blanquet,S. and Mechulam,Y. (1995) Transition state stabilisation by the 'high' motif of class I aminoacyl-tRNA synthetases: the case of Escherichia coli methionyl-tRNA synthetase. Nucleic Acids Res., 23, 4793-4798.

I thought maybe the journal has changed style, but in the latest issue I'm seeing
image

So also the old formatting of this style. First, could we make sure we're talking about the same journal, i.e. https://academic.oup.com/nar . If so, could you check back with the editor what's going on? Is that a changed requirement? Is it documented somewhere?

**Comment by jimhavrilla** [comment]
This is incorrect.  If you go to any new NAR article you can see that is
not the citation style.  And we just got an article accepted there and the
editor told us our style, this old style, is wrong.
https://academic.oup.com/nar/article/48/8/3999/5809668#203078834 is a new
article for the may 07 issue in advance press, so the most up-to-date
possible.  And you can see the citation style looks as I have programmed it:
[image: image.png]

**Comment by adam3smith** [comment]
No, check the PDF of that article -- it uses what you call the old style.  HTML citation styles often don't match journal styles, so the different style there is not saying much. 

Since we have current PDFs as well as current instructions for authors pointing to the style as it is, I would kindly ask you to check back with the editor what's going on and if it would be possible to publish/point to a document describing up-to-date style requirement, should they really have changed.

**Comment by jimhavrilla** [comment]
I see what you mean.  It's strange, since the editor literally wrote us two
days ago...now this was for NAR's new sister journal NARGAB (Nucleic Acid
Research -Genomics and Bioinformatics) which is an online journal only.
And since (I think) NAR still has a print version, maybe main NAR does not
care?  Can we potentially make a new CSL of these edits for NARGAB?

>

**Comment by rmzelle** [comment]
> this was for NAR's new sister journal NARGAB (Nucleic Acid Research -Genomics and Bioinformatics)

@adam3smith, see https://academic.oup.com/nargab/pages/General_Instructions:

> **Manuscript format, structure, and style**
> The journal follows the manuscript preparation conventions as Nucleic Acids Research

> **References**
> * May be formatted in any readable style at submission

`¯\_(ツ)_/¯`

**Comment by jimhavrilla** [comment]
Submission, yes. Not acceptance. That is the stage they asked me to change
it.

Jim Havrilla

**Comment by rmzelle** [comment]
> Submission, yes. Not acceptance. That is the stage they asked me to change it.

Did they point you to a webpage with the desired citation format, then? I can't find anything else on the journal's website.

**Comment by jimhavrilla** [comment]
Nope, basically just told me this, and then I had to look up the HTML based
format...there doesn't seem to be anything specifying it, it's quite
frustrating.

**Comment by rmzelle** [comment]
Since we don't have a CSL style for "NAR Genomics and Bioinformatics", and there are no public guidelines for this journal, I'm not sure we can do anything here. Sorry we can't be more helpful.

**Comment by jimhavrilla** [comment]
Eh, not an issue for me, I have the CSL saved.  Was just trying to be
potentially helpful to others, but I wouldn't recommend using this style
for NARGAB in the future without EndNote export so they can correct it for
you.

**Comment by rmzelle** [comment]
(PDFs in the most recent issue of this journal like https://academic.oup.com/nargab/article/2/1/lqz005/5576141 also don't use `sort-separator=" "`:

1. Biffi,G., Tannahill,D., McCafferty,J. and Balasubramanian,S. (2013) Quantitative visualization of DNA G-quadruplex structures in human cells.Nat. Chem.,5, 182–186.

)

**Pull request closed without merging**